src/components/share/partials/share.html

```diff
@@ -28,6 +28,12 @@
     href="https://plus.google.com/share?url={{encodedPermalinkHref}}">
     <i class="fa fa-google-plus"></i>
   </a>
+
+  <a ng-if="showWhatsapp" class="ga-share-icon"
+    data-original-title="{{'whatsapp_tooltip' | translate}}"
+    href="whatsapp://send?text={{permalinkValue}}">
```

Comment:
Shouldn't we use the `shortenUrl`  here ? The link is very big on the message

Reply:
That s the shorten value. Try it out on your phone